widgets/calendarcombo/org.eclipse.nebula.widgets.calendarcombo/src/org/eclipse/nebula/widgets/calendarcombo/CustomCombo.java

```diff
@@ -259,11 +258,7 @@ public void add(String string, int index) {
 	 * @see #removeModifyListener
 	 */
 	public void addModifyListener(ModifyListener listener) {
-		checkWidget();
```

Comment:
Why is the checkwidget and the null check removed?

Reply:
`Widget.addTypedListener()` does the same already, so it would just be a duplication to do it here.